test/e2e/config/cloudstack.yaml

```diff
@@ -134,6 +134,7 @@ variables:
   CLOUDSTACK_IMPOSSIBLE_CONTROL_PLANE_MACHINE_OFFERING: "Impossible Instance"
   CLOUDSTACK_WORKER_MACHINE_OFFERING: "Medium Instance"
   CLOUDSTACK_IMPOSSIBLE_WORKER_MACHINE_OFFERING: "Impossible Instance"
+  CLOUDSTACK_ETCD_MACHINE_OFFERING: "Large Instance"
```

Comment:
why do we need this?

Reply:
it is not referred anymore. During previous commit, it is used to set up an etcd CSMachine offering.